scripts/run-indexer.sh

```diff
@@ -193,7 +193,7 @@ runIndexer () {
         _CURRENT_BRANCH=$(git branch | sed -n -e 's/^\* \(.*\)/\1/p')
         if [[ "$_GITBRANCH" != "$_CURRENT_BRANCH" ]]; then
             echo "Your git branch [${_GITBRANCH}] parameter does not match the selected git branch in the project [${_CURRENT_BRANCH}]"
-            exit 1
+        #    exit 1
```

Comment:
This could have an impact on other people who use the script outside of Jenkins. You might want to talk to @gsviteri about this one.

Reply:
Yea, fair point -- this branch can remain open until that discussion takes place

Reply:
Hi @cookersjs and @SolomonShorser-OICR 

By default we assume it is master branch, however if the local copy is point to a different branch, there might be prune to errors. That's why we stop the execution and then you can address it manually.

Is the $_CURRENT_BRANCH a problem for Jenkins ?  we can workaround that and this if statement pass if it is "jenkins" ...

Reply:
Actually, its a bit of a strange situation - Jenkins clones a repository but then takes a HEAD of the current branch, which is why it is flagged by this program. I have been thinking recently about another solution - rather than have the Jenkinsfile for the search-indexer in this repository, I might just move the JF to [release-jenkins-utils/jenkins](https://github.com/reactome/release-jenkins-utils/tree/develop/jenkins), and then specify the JF to clone the default search-indexer repository. 

This approach would get around the branch check in this file, as it won't be taking a HEAD of the branch anymore. This goes along with another idea I had recently, which is that it might be good in the future to move all Jenkinsfiles to a single repository (release-jenkins-utils) for easier management of the system. Right now we have a couple of JFs at that location, for steps that aren't easily placed in a single repository, or for those that haven't been converted to Java.

That was a bit of a paragraph! All I ask right now is that this branch stays open until I've implemented that change. I'll clean this up before I'm finished next week!

Reply:
Perfect! Understood.
Please let me know if you want to me to do anything !